src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -504,6 +505,50 @@ public bool IsIronPythonDialogDisabled
             }
         }
 
+        //private ICSharpCode.AvalonEdit.TextEditorOptions editTextOptions = new ICSharpCode.AvalonEdit.TextEditorOptions() {
+        //    ConvertTabsToSpaces = false
+        //};
+        private DynamoTextOptions editTextOptions = new DynamoTextOptions();
+
+        /// <summary>
+        /// Gets the text editor options for python script editor.
+        /// </summary>
+        public DynamoTextOptions TextOptions
+        {
+            get
+            {
+                return editTextOptions;
+            }
+
+            set
+            {
+                if (editTextOptions != value)
+                {
+                    editTextOptions = value;
+                    RaisePropertyChanged(nameof(TextOptions));
+                }
+            }
+        }
+
+        /// <summary>
+        /// Indicates if whether the whitespaces and tabs should be visible in the python script editor.
+        /// </summary>
+        public bool ShowTabsAndSpacesInScriptEditor
+        {
+            get
+            {
+                return model.PreferenceSettings.ShowTabsAndSpacesInScriptEditor;
+            }
+            set
+            {
+                TextOptions.SetTextOptions(value);
+                model.PreferenceSettings.ShowTabsAndSpacesInScriptEditor = value;
+                RaisePropertyChanged(nameof(ShowTabsAndSpacesInScriptEditor));
```

Comment:
because both @mmisol  and I were confused when looking at this initially, maybe it is worth adding a comment that this property is for the menu item?